docs/deployment/landing_page.rst

```diff
@@ -3,6 +3,19 @@
 Landing Page
 ============
 
+SecureDrop itself is located on a Tor hidden service. Organizations also need to
```

Comment:
I feel like a better way to phrase this is "SecureDrop runs as a Tor Hidden Service" or "SecureDrop is a Tor Hidden Service." Something about "is located on a" sounds off to my ear. But also I don't even know how to English any more, so additional input from the team would be helpful. :)